chapters/en/chapter10/4.mdx

```diff
@@ -0,0 +1,343 @@
+
+Now that we’ve defined our task, we can begin looking into speech recognition in more detail. Specifically, we’ll cover:
+* How to choose a dataset
+* How to load a dataset
+* What models we can use
+* How to prepare audio-text data
+* Metrics for speech recognition
+* How to fine-tune an ASR system with the Trainer API
+
+### How to choose a dataset
+As with any machine learning problem, our model is only as good as the data that we train it on. Speech recognition datasets vary considerably in how they are curated and the domains that they cover. To pick the right dataset, we need to match our criteria with the features that a dataset offers.
+
+Before we pick a dataset, we first need to understand some of the key defining features.
+
+#### 1. Number of hours
+Simply put, the number of training hours indicates how large the dataset is. It’s analogous to the number of training examples in an NLP dataset. However, bigger datasets aren’t necessarily better. If we want a model that generalises well, we want a diverse dataset with lots of different speakers, domains and speaking styles.
+
+#### 2. Domain
+The domain entails where the data was sourced from, whether it be audiobooks, podcasts, YouTube or financial meetings. Each domain has a different distribution of data. For example, audiobooks are recorded in high-quality studio conditions (with no background noise) and text that is taken from written literature. Whereas for YouTube, the audio likely contains more background noise and a more informal style of writing.
+
+We need to match our domain to the conditions we anticipate at inference time. For instance, if we train our model on audiobooks, we can’t expect our model to perform well in noisy environments.
+
+#### 3. Speaking style
+The speaking style falls into one of two categories:
+* Narrated: read from a script
+* Spontaneous: un-scripted, conversational speech
+
+The audio and text data reflect the style of speaking. Since narrated text is scripted, it tends to be spoken articulately and without any errors:
+
+“Consider the task of training a model on a speech recognition dataset”
+
+Whereas for spontaneous speech, we can expect a more colloquial style of speech, with the inclusion of repetitions, hesitations and false-starts:
+
+“Let’s <uhm> take a look at how <silence> you’d you’d go about training a model on <uhm> a sp- speech recognition dataset” (this is an extreme example)
+
+#### 4. Transcription style
+Whether the text has punctuation, casing or both. If we want a system to generate fully formatted text that could be used for a publication or meeting transcription, we require training data with punctuation and casing. If we just require the spoken words in an unformatted structure, neither punctuation nor casing are necessary.
+
+A summary of datasets on the HF Hub:
```

Comment:
```suggestion
Here is a summary of popular speech recognition datasets on the Hugging Face Hub:
```

Reply:
Resolved in https://github.com/huggingface/course/pull/294/commits/9d111104b597c83c0eb05f25f36544eed01eb3a0